anaconda_build/README.md

```diff
@@ -73,3 +73,58 @@ local development. E.g. on uframe-3-test, mi-instrument is installed under the r
 ```bash
 $ conda install mi-instrument
 ```
+
+## The stream/ingest engines and environment
```

Comment:
the building of the stream_engine, ingest_engine and enginemeta packages shouldn't be drastically different from how the other packages are built, if it is, then we probably are not using the build system correctly.

Reply:
Agreed. They are much more in line with them now.